### ISSUE TITLE
RegExp DoS issue due to Wrong dependency version resolved

### ISSUE THREAD

**Issue by santiph** [gunpowderlabs/gulp-environments#9]
`npm install gulp-environments@0.1.1` installs `gulp-if@1.2.5`, which uses minimatch@1.0.0. Producing the following error message:
`npm WARN deprecated minimatch@1.0.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue`

**Comment by santiph** [comment]
According to https://github.com/gunpowderlabs/gulp-environments/blob/master/package.json
gulp-if 2.0.1 should be used, Why am I getting 1.2.5 instead? 
```
├─┬ gulp-environments@0.1.1
│ └─┬ gulp-if@1.2.5
│   └─┬ gulp-match@0.2.1
│     └── minimatch@1.0.0 
```

**Comment by wrozka** [comment]
I've just published v0.1.2, it includes the dependency updates, please give it a try.

**Comment by santiph** [comment]
Perfect! Thanks a lot for the quick response. ;)
